bin/ycsb

```diff
@@ -103,6 +105,30 @@ def get_ycsb_home():
         dir = os.path.join(dir, os.path.pardir)
     return os.path.abspath(dir)
 
+def is_distribution():
+    # If there's a top level pom, we're a source checkout. otherwise a dist artifact
+    return "pom.xml" not in os.listdir(get_ycsb_home())
+
+# Run the maven dependency plugin to get the local jar paths.
+# presumes maven can run, so should only be run on source checkouts
+# will invoke the 'package' goal for the given binding in order to resolve intra-project deps
+# presumes maven properly handles system-specific path separators
+def get_classpath_from_maven(database):
+    DEVNULL = open(os.devnull, 'w')
+    mvn = subprocess.Popen(["mvn", "-pl", "com.yahoo.ycsb:"+database+"-binding", "-am",
```

Comment:
It will work on cygwin. We have 1 open ticket of someone trying to run on bare windows without cygwin, but I'm not sure if we actually work in that environment as is.

I could switch to munging the output here, but it'll take a bit. Maybe we could make it a follow on?